archives/archives_test.go

```diff
@@ -37,6 +37,9 @@ func TestGetMissingDayArchives(t *testing.T) {
 	// get the tasks for our org
 	ctx := context.Background()
 	config := NewDefaultConfig()
+	config.AWSAccessKeyID = "missing_aws_access_key_id"
+	config.AWSSecretAccessKey = "missing_aws_secret_access_key"
```

Comment:
do these have to be set for these tests?

Reply:
apparently yes. They were the old defaults and the tests fail without them